Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ruby:2.2.3
+
+RUN apt-get update -qq && apt-get install -y build-essential
```

Comment:
You should group all of these together to make the image smaller so something like this

```
RUN apt-get update -qq && apt-get install -y build-essential libpq-dev libxml2-dev libxslt1-dev libqt4-webkit libqt4-dev xvfb nodejs && apt-get clean
```

The reason for this is to minimize the layers so the docker container is smaller